test/__snapshots__/exampleGeneratorTests.ts.snap

```diff
@@ -125,6 +125,85 @@ Array [
       "source": "response",
     },
   ],
+  Object {
+    "code": "INVALID_FORMAT",
+    "exampleJsonPath": "$parameters.parameters",
+    "examplePosition": Object {
+      "column": 19,
+      "line": 8,
+    },
+    "exampleUrl": "test/exampleGenerator/specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/ManagedDatabaseCompleteExternalRestore.json",
+    "message": "Object didn't pass validation for format uri: testdb1_log4",
```

Comment:
Can you share why test still doesn't pass URL format validation?

Reply:
Oh, I forgot update the example file.